openshift/ocp4/matomo/openshift/templates/matomo/matomo-deploy.json

```diff
@@ -348,7 +348,7 @@
       "displayName": "Resources CPU Limit",
       "description": "The resources CPU limit (in cores) for this build.",
       "required": true,
-      "value": "250m"
+      "value": "500m"
```

Comment:
Is there a reason that we're increasing the limit and not lowering the request here?

Reply:
Yes, originally the solution was to reduce the CPU-request to 50, and the CPU-limit would remain at 250m. These settings caused a delayed restart on the pod. A recommended solution was to increase the CPU-request to 100m, and increase the CPU-limit to 500m(a level account for spikes from a request of 100m). 

The reduced CPU-Request will give the desired outcome.